### DISCUSSION TITLE
Column Groups

### DISCUSSION THREAD

**Discussion by dnjsgur0629** [KevinVandy/material-react-table#234] [Ideas]
Thanks for providing this library.
I'm looking forward to adding column groups feature.
<br/>
like this:
![image](https://user-images.githubusercontent.com/65391273/206093229-eb1e7e2b-71fd-4ade-8b0f-a00dc82ebfb6.png)
<br/>
A nested column like this will be of great help to junior developers like me.
And if there is any other way to achieve this, could you please let me know?
[I found this here](https://tanstack.com/table/v8/docs/examples/react/column-groups)

**Comment by KevinVandy** [comment]
MRT implements pretty much all of the TanStack Table features, especially column groups. They are present in the advanced example.
https://www.material-react-table.com/docs/examples/advanced
